content/learn/book/getting-started/setup/_index.md

```diff
@@ -105,6 +104,15 @@ edition = "2018"
 bevy = "0.1.2" # make sure this is the latest version
 ```
 
+### Make debug builds faster
+
+Debug Rust builds are notoriously slow, and this becomes an issue when making games. Luckily, you can set up rust so your dependendencies (including Bevy) are compiled with optimizations enabled, giving you faster runtime speeds without sacrificing debuggability or iteration speed with the ```--release``` flag. To enable this, append the following to your Cargo.toml file:
+
+```toml
+[profile.dev.packages."*"]
+opt-level = 3
+```
+
 Run ```cargo run``` again. The Bevy dependencies should start building. This will take some time as you are essentially building an engine from scratch. You will only need to do a full rebuild once. Every build after this one will be fast!
```

Comment:
There’s no heading above this paragraph which might suggest it’s still part of the previous section. Maybe we should move the `make debug builds faster` section somewhere else. Maybe @cart can suggest a better place.

Reply:
There wasn't a title beforehand either, but perhaps there should. I'm open to suggestions